src/screencapturekit.cs

```diff
@@ -335,4 +453,105 @@ interface SCStreamOutput {
 		[Export ("stream:didOutputSampleBuffer:ofType:")]
 		void DidOutputSampleBuffer (SCStream stream, CMSampleBuffer sampleBuffer, SCStreamOutputType type);
 	}
+
+	[NoiOS, NoTV, NoWatch, Mac (14, 0), NoMacCatalyst]
+	[BaseType (typeof (NSObject))]
+	interface SCContentSharingPickerConfiguration {
+		[Export ("allowedPickerModes", ArgumentSemantic.Assign)]
+		SCContentSharingPickerMode AllowedPickerModes { get; set; }
+
+		[Export ("excludedWindowIDs", ArgumentSemantic.Strong)]
+		NSNumber [] ExcludedWindowIDs { get; set; }
```

Comment:
```suggestion
		NSNumber [] ExcludedWindowIds { get; set; }
```
I think the team decided on "Id" instead of "ID"